app/src/main/res/values-az/strings.xml

```diff
@@ -31,7 +31,7 @@
     <string name="account_settings_subscribed">Abunə olundu</string>
     <string name="addBookmark">Əlfəcinlərə əlavə et</string>
     <string name="app_bars_dot_spacer">·</string>
-    <string name="app_name">Jerboa</string>
+    <string name="app_name" translatable>Jerboa</string>
```

Comment:
No, I mean the whole string. When a entry isn't found in a translation it falls back to the default. So it isn't needed here. Same with all the English entries.